dashboard/app/models/levels/pythonlab.rb

```diff
@@ -78,4 +77,24 @@ def clean_up_predict_settings
       predict_settings.delete("multipleChoiceOptions")
     end
   end
+
+  # Return the validation condition for this level. If the level has a validation file, the condition
+  # is that all tests passed. If there is no validation file, there are no conditions.
+  def get_validations
+    has_validation = start_sources && start_sources["files"]&.any? {|(_, file)| file["type"] == 'validation'}
+    if has_validation
+      [{
+        conditions: [
+          {
+            name: 'PASSED_ALL_TESTS',
```

Comment:
Out of curiosity, are there plans to eventually move this into levelbuilder with multiple conditions (ex. `PASSED_FIRST_TEST`, `PASSED_SOME_TESTS`, etc)?

Reply:
Not currently planned, I think we will always want them to pass all tests.